crates/biome_css_analyze/src/lint/nursery/no_invalid_direction_in_linear_gradient.rs

```diff
@@ -50,39 +50,20 @@ declare_rule! {
 
 lazy_static! {
     // It is necessary to find case-insensitive string.
-    // For example, both 'linear-gradinet' and 'Linear-gradient' should pass the check.
-    pub static ref LINEAR_GRADIENT_FUNCTION_NAME: Regex =
-        Regex::new(r"^(?i)(-webkit-|-moz-|-o-|-ms-)?linear-gradient").unwrap();
-
-    // It is necessary to find case-insensitive string.
-    // Also Check if 'in' is a word.
+    // Also Check if 'in' is a word boundary.
     // For examples,`to top in srgb` is valid but `to top insrgb` is not valid.
     pub static ref IN_KEYWORD: Regex = Regex::new(r"(?i)\bin\b").unwrap();
 
     // This regex checks if a string consists of a number immediately followed by a unit, with no space between them.
+    // For examples, `45deg`, `45grad` is valid but `45 deg`, `45de` is not valid.
     pub static ref ANGLE: Regex = Regex::new(r"^[\d.]+(?:deg|grad|rad|turn)$").unwrap();
```

Comment:
This regular expression checks if the value is provided in a valid angle format.

> The <angle> data type consists of a <number> followed by one of the units listed below. As with all dimensions, there is no space between the unit literal and the number. The angle unit is optional after the number 0.
> https://developer.mozilla.org/en-US/docs/Web/CSS/angle

**examples
valid**

- 4deg
- 45deg
- 135deg

**invalid**

- 4 deg
- 45 de
- deg